TypeScript/recursion.ts

```diff
@@ -0,0 +1,7 @@
+const rec = (n: number) => {
+  let i = n;
+  console.log(i);
+  rec(++i);
+}
```

Comment:
Would be better to have it as a function instead of arrow function. Also did you check the `++i` syntax works correctly?

```suggestion
function rec (n: number) {
  let i = n;
  console.log(i);
  rec(++i);
}
```

Reply:
Using normal function is for normies, and yes it works.

Reply:
> Using normal function is for normies, and yes it works.

That's the standards